provisioner/aws-manager.js

```diff
@@ -129,6 +134,32 @@ AwsManager.prototype._filterSpotRequests = function(spotReqs) {
   return data;
 };
 
+
+/**
+ * Instead of defining these lists in lots of places, let's ensure
+ * that we use the same filters in all places!
+ */
+
+AwsManager.prototype.filters = {
```

Comment:
I purposely don't have any sort of abstraction on the api objects as I don't really want to maintain an abstraction on top of the ec2 api.  I'm relying on the API to return properly structured objects, which it does in every case that I've tried.  We actually will get exceptions thrown in the method that reads the Instances and Requests in if they do not contain all of the keys that we want, so I'm not too concerned about fields being missing.

As for storing the filters as a global constant, sure.  I don't see a reason to not expose them.